### ISSUE TITLE
additional coloring on runs or sensors

### ISSUE THREAD

**Issue by bargheertarje** [dagster-io/dagster#17674]
### What's the use case?

I was wondering if it would be sensible to have more color-coding on the list of running schedules and sensors. At the moment they are all green or grey, which indicates running or deactivated. How about coloring them red when they are running, but failing?

If for instance a schedule have somehow eluded coverage in unit-tests and are starting to fail, I will only notice it by either clicking on the schedule (which takes some clicks to get to) - or having runs not appear when they are scheduled.

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by sryza** [comment]
Thanks for the request @bargheertarje. Forwarding this to our UI team.

**Comment by hellendag** [comment]
Hi @bargheertarje, can you provide a screenshot of which pieces of the UI you're referring to? Thanks!

**Comment by benoitsdm** [comment]
I agree more color would be useful! In my case, I would like to have a different color when a job run retry is failed, and when a run fail the first time but the retry is success (something like orange). 
It would be much easier to see in the overview if something is good, or if a retry is has failed and an intervention is needed.
An API request is often failing but when the retry is a success, i would like to see it.
